lib/exqlite.ex

```diff
@@ -3,102 +3,24 @@ defmodule Exqlite do
   SQLite3 driver for Elixir.
   """
 
-  alias Exqlite.Connection
-  alias Exqlite.Error
-  alias Exqlite.Query
+  alias Exqlite.RWConnection
   alias Exqlite.Result
 
-  @doc "See `Exqlite.Connection.connect/1`"
-  @spec start_link([Connection.connection_opt()]) :: {:ok, pid()} | {:error, Error.t()}
-  def start_link(opts) do
-    DBConnection.start_link(Connection, opts)
-  end
```

Comment:
Most of the public API would need to be restored before merging if this experiment is successful.